mix.exs

```diff
@@ -40,7 +40,7 @@ defmodule GRPC.Mixfile do
       {:jason, ">= 0.0.0", optional: true},
       {:cowlib, "~> 2.12"},
       {:castore, "~> 0.1 or ~> 1.0", optional: true},
-      {:protobuf, "~> 0.11"},
+      {:protobuf, "~> 0.11", only: [:dev, :test]},
```

Comment:
I think this has already been changed before.

Can you check the history to confirm, please?

Simplest solution for you is to override: true on this dependency

Reply:
I think can't `override: true` because I'm a package too :(

Will double check the history

I'd understand if its not accepted (but then the readme would need to be updated)

Reply:
@polvalente #233

Reply:
@sleipnir was the change intentional? 👀

Reply:
Looks like it breaks without this dependency, because of some new files that were added that use `:protobuf` in comp time...

Reply:
@v0idpwn Yes for implementing http transcoding we had to add some things, I'm not sure if it's possible to do without this. I may look into this in the future and try to reopen this issue if I see a solution.